torchvision/utils.py

```diff
@@ -3,7 +3,7 @@
 irange = range
 
 
-def make_grid(tensor, nrow=8, padding=2,
+def make_grid(tensor, nrow=8, padding=2, pad_value=0,
```

Comment:
While I prefer this order of arguments, do we want to possibly break backward compatibility by changing the order of arguments?
Also, the README is not automatically updated it seems. @soumith do we need to modify by hand both the inline documentation and the README?